src/main.scss

```diff
@@ -80,41 +91,85 @@ $dark: #212529;
 }
 .addBtn {
   background: linear-gradient(180deg, var(--addButton) 0%, var(--addButton2) 85%, var(--addButton3) 100%);
+  border: hidden;
+  border-radius: 5px;
+  text-align: center;
+  line-height: 30px;
 }
 .addBtn:active {
   transform: scale(0.9);
 }
 .shoulder {
+  display: flex;
```

Comment:
use scss mixins for these classes, as they are exactly the same except for the class name and the variable used

Reply:
done